code/mission/missiongoals.cpp

```diff
@@ -1522,3 +1551,23 @@ int mission_event_find_sexp_tree(int root_node)
 			return i;
 	return -1;
 }
+
+int mission_goal_lookup(const char *name)
+{
+	Assertion(name != nullptr, "Goal name cannot be null!");
+
+	for (int i = 0; i < Num_goals; ++i)
+		if (!stricmp(Mission_goals[i].name, name))
+			return i;
+	return -1;
+}
+
+int mission_event_lookup(const char *name)
+{
+	Assertion(name != nullptr, "Event name cannot be null!");
+
+	for (int i = 0; i < Num_mission_events; ++i)
+		if (!stricmp(Mission_events[i].name, name))
+			return i;
+	return -1;
+}
```

Comment:
What are these added for?
Remnants of a different PR maybe?

Reply:
These are in preparation for #4634, but I can move them to that PR if you think I should.

Reply:
eh, its fine in that case. I was just wondering